src/actions/tutorialPrev.ts

```diff
@@ -8,7 +8,7 @@ import getSetting from '../selectors/getSetting'
 const tutorialPrev = (state: State, { hint }: { hint?: boolean } = {}) => {
   // @typescript-eslint/eslint-plugin does not yet support no-extra-parens with nullish coallescing operator
   // See: https://github.com/typescript-eslint/typescript-eslint/issues/1052
-  // eslint-disable-next-line @typescript-eslint/no-extra-parens
```

Comment:
?

Reply:
Basically as per eslint v9.

if rule define is in eslint.config.ts and rule is set to 0 or off. and we have set the commad in code so eslint lint will show that command as warning.

so we just fix all these by running yarn lint --fix. if you want to keep these warning as it is so will revert all these.

Reply:
rule for no-extra-parens is define in eslint config and set to off same as your previous eslint config. so we dont need to add this line here anymore. because of this lint showing just warning.